xbmc/video/ContextMenus.cpp

```diff
@@ -119,7 +120,17 @@ static void SetPathAndPlay(CFileItem& item)
     item.SetPath(item.GetVideoInfoTag()->m_strFileNameAndPath);
   }
   item.SetProperty("check_resume", false);
-  g_application.PlayMedia(item, "", PLAYLIST_NONE);
+
+  CFileItemPtr movieItem((std::make_shared<CFileItem>(item)));
```

Comment:
I would suggest to write <code>const CFileItemPtr movieItem(new CFileItem>(item));</code> here. "const as const can" and less letters to type for the object creation.